stats/gRPC.md

```diff
@@ -66,6 +69,11 @@ The following set of views are considered minimum required to monitor client-sid
 | grpc.io/client/roundtrip_latency      | grpc.io/client/roundtrip_latency      | distribution | grpc_client_method                     |
 | grpc.io/client/completed_rpcs         | grpc.io/client/roundtrip_latency      | count        | grpc_client_method, grpc_client_status |
 | grpc.io/client/started_rpcs           | grpc.io/client/started_rpcs           | count        | grpc_client_method                     |
+| grpc.io/client/sent_message_bytes     | grpc.io/client/sent_message_bytes     | distribution | grpc_client_method                     |
+| grpc.io/client/received_message_bytes | grpc.io/client/received_message_bytes | distribution | grpc_client_method                     |
+| grpc.io/client/sent_message_count     | grpc.io/client/sent_message_bytes     | count        | grpc_client_method                     |
```

Comment:
Should be referred as "count", not "bytes".

Reply:
This has the same rational to https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/gRPC.md#why-are-completed_rpcs-views-defined-over-latency-measures. We can calculate the message count by how many times "sent_message_bytes" are recorded.

Reply:
yes, I will remove the _count views, since the count will be in the distribution of the _bytes views